challenge.rb

```diff
@@ -1,25 +1,25 @@
 def capitalize_each_string(input)
-  return input.map{|element| element.capitalize}
+  input.map{|element| element.capitalize}
```

Comment:
This is legit, but most folks would write it as: `input.map(&:capitalize)`. Google "ruby symbol to_proc" for more info. Its kinda hard to explain.